src/blurredLocation.js

```diff
@@ -8,13 +8,21 @@ BlurredLocation = function BlurredLocation(options) {
   options.pixels = options.pixels || 400;
 
   options.gridSystem = options.gridSystem || require('./core/gridSystem.js');
+  options.Interface = options.Interface || require('./ui/Interface.js');
 
   gridSystemOptions = options.gridSystemOptions || {};
   gridSystemOptions.map = options.map;
   gridSystemOptions.gridWidthInPixels = gridWidthInPixels;
   gridSystemOptions.getMinimumGridWidth = getMinimumGridWidth;
   gridSystem = options.gridSystem(gridSystemOptions);
 
+  InterfaceOptions = options.InterfaceOptions || {};
+  InterfaceOptions.Id = {latId: 'lat', lngId: 'lng'}
```

Comment:
Its a good question. How about: directly manipulating the Leaflet map is not considered part of our UI -- leaflet has its own UI. Our UI is any part of the DOM we created and are managing directly.